Spigot-Server-Patches/0543-Fix-SPIGOT-5824-Bukkit-world-container-is-not-used.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: William Blake Galbreath <Blake.Galbreath@GMail.com>
+Date: Fri, 10 Jul 2020 13:12:33 -0500
+Subject: [PATCH] Fix SPIGOT-5824 Bukkit world-container is not used
+
+
+diff --git a/src/main/java/net/minecraft/server/Main.java b/src/main/java/net/minecraft/server/Main.java
+index 2f29d7040d37e8be2ba5f8f634011e069a07db2f..d5ae5c9da76134cfa0df125fb96db67071ed77d2 100644
+--- a/src/main/java/net/minecraft/server/Main.java
++++ b/src/main/java/net/minecraft/server/Main.java
+@@ -86,7 +86,20 @@ public class Main {
+                 return;
+             }
+ 
+-            File file = (File) optionset.valueOf("universe"); // CraftBukkit
++            // Paper start - fix SPIGOT-5824
++            File file;
++            if (optionset.has("universe")) {
```

Comment:
Are you sure this isn't always `true`? `universe` is set to return a File by default.

Reply:
Yes, `has` doesn't check the defaults, just what's `detected` to actually be set.